#global definition 
std_msgs/String filename_reference
std_msgs/String filename_target
geometry_msgs/Point point_offset
std_msgs/Float32 theta_rad
std_msgs/UInt32 focuspos
geometry_msgs/Polygon[] roi_polygons
std_msgs/Float32 scale_factor_x
std_msgs/Float32 scale_factor_y
---
#result definition
geometry_msgs/Point point_result
---
#feedback
std_msgs/Byte currentState
geometry_msgs/Point currentPosition
